src/components/UI/FormFields/InputField/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { TextField, TextFieldProps } from "@mui/material";
+import { FieldError } from "../FieldError";
+
+interface IProps {
+  defaultProps: TextFieldProps;
+  field: any;
+  isError: boolean;
+  error: string;
+}
+
+export const InputField = ({ field, isError, error, defaultProps }: IProps) => {
+  return (
+    <>
+      <TextField error={isError} type="text" {...field} {...defaultProps} />
+      <FieldError>{isError ? error : ""}&nbsp;</FieldError>
```

Comment:
Why `&nbsp;`

Reply:
without it it doesnt render the element for some reason because its empty